src/pkg/bundle/inspect.go

```diff
@@ -5,13 +5,48 @@
 package bundle
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/defenseunicorns/pkg/oci"
 	"github.com/defenseunicorns/uds-cli/src/config"
 	"github.com/defenseunicorns/uds-cli/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/pkg/layout"
+	"github.com/defenseunicorns/zarf/src/pkg/packager/filters"
+	zarfSources "github.com/defenseunicorns/zarf/src/pkg/packager/sources"
 	zarfUtils "github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/pkg/zoci"
+	zarfTypes "github.com/defenseunicorns/zarf/src/types"
+	"github.com/fatih/color"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pterm/pterm"
 )
 
 // Inspect pulls/unpacks a bundle's metadata and shows it
 func (b *Bundle) Inspect() error {
+	//  handle --list-images flag
+	if b.cfg.InspectOpts.ListImages {
+		err := utils.CheckYAMLSourcePath(b.cfg.InspectOpts.Source)
+		if err != nil {
```

Comment:
nit, but is there a particular reason to have the err check separate on this one but "inlined" (if that's the right word) for the next call. just wondering if we can make them all look the same for consistency.

Also, do you think there's any value in extracting the list image functions into a separate function `listImages()` that gets called in `Inspect()`. Feels like it's getting a little long.